env/nginx/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM nginx:1.9
-MAINTAINER Nicolas Ruflin <spam@ruflin.com>
+FROM nginx
```

Comment:
I prefer to have a specific version in here instead of latest. I had the experience in the past that with latest a new version gets pulled automatically which can break the CI. As I want tests to be reproducible, a fixed version should be kept in.